lib/apachetest.pm

```diff
@@ -334,4 +333,19 @@ qq{mysql -u root -e "CREATE DATABASE openQAdb; USE openQAdb; CREATE TABLE test (
     assert_script_run qq{mysql -u root -e "DROP DATABASE openQAdb;"};
 }
 
+# poo#62000
+sub postgresql_cleanup {
+    select_console 'root-console';
+    systemctl 'stop postgresql';
+    zypper_call 'in lsof';
+    systemctl 'disable postgresql';
+    assert_script_run q{kill -s KILL $(ps -u postgres -o pid=)};
```

Comment:
wow, you really hate postgresql that much?... stopping and disabling postgres should be enough. If postmaster is still alive, it might be due to an open connection?.

Reply:
:D well, the problem is disc space, thus I guess we need to get rid of it completely. No mercy allowed over here.